v2/http.go

```diff
@@ -75,7 +70,7 @@ func Server(home string) http.Handler {
 // to correlate.
 func errorHandler(logger *logrus.Logger) func(context.Context, http.ResponseWriter, error) {
 	return func(ctx context.Context, w http.ResponseWriter, err error) {
-		id := ctx.Value(middleware.RequestIDKey).(string)
+		id := ctx.Value(middleware.RequestID).(string)
 		w.Write([]byte("[" + id + "] encoding: " + err.Error()))
 		logger.Printf("[%s] ERROR: %s", id, err.Error())
```

Comment:
```suggestion
id := ctx.Value(goamiddleware.RequestIDKey).(string)
```

the `ctx.Value` takes in `any` and returns `any`. The problem is that we're passing the wrong argument. After digging a bit we have to import:
 `goamiddleware "goa.design/goa/v3/middleware"`
 which exposes the context key that is used to store ID info. (This is the same context key used by the `RequestID` middleware)